src/components/Book.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/destructuring-assignment */
 /* eslint-disable react/prop-types */
 /* eslint-disable no-restricted-syntax */
 /* eslint-disable react/prefer-stateless-function */
```

Comment:
Let's resolve this disabled rules
- [`react/destructuring-assignment`] Let's destructure the props, we could do 
```
const { book: { id, title, category } } = this.props;
```
- [`react/prop-types`] Please check out https://reactjs.org/docs/typechecking-with-proptypes.html to typecheck your props

- [`react/prefer-stateless-function`] you can use a stateless function for this component inside of a class component

Reply:
@grace-mugoiri, Good job making some corrections.
Kindly make the remaining few ones from the previous reviewer @bolah2009
I quote: 
```
[react/prefer-stateless-function] you can use a stateless function for this component inside of a class component.
```
```
[react/prop-types] Please check out https://reactjs.org/docs/typechecking-with-proptypes.html to type check your props.
```